shared/reducers/chat2.tsx

```diff
@@ -619,12 +619,24 @@ const reducer = Container.makeReducer<Actions, Types.State>(initialState, {
     const unreadMap = new Map<Types.ConversationIDKey, number>()
     conversations.forEach(({convID, badgeCounts, unreadMessages}) => {
       const key = Types.conversationIDToKey(convID)
-      const count = badgeCounts[badgeKey] || 0
+      // if its big we treat the count as 1 or 0 so additional chats don't redraw us
```

Comment:
The mobile convo header uses `badgeMap` to show a badge by the back button, with this change it won't show more messages past the first. Maybe we could just do this memoized in the inbox connector?

Reply:
the header should just show whats in the nav and not compute it again

Reply:
It excludes badges for the current chat. otherwise I think the badge flickers in as you're looking at the thread